concordium-sdk/src/main/java/com/concordium/sdk/responses/blockitemsummary/AccountTransactionDetails.java

```diff
@@ -1,22 +1,23 @@
 package com.concordium.sdk.responses.blockitemsummary;
 
-import com.concordium.grpc.v2.AccountTransactionEffects;
+import com.concordium.sdk.crypto.bls.BLSPublicKey;
+import com.concordium.sdk.responses.BakerId;
 import com.concordium.sdk.responses.modulelist.ModuleRef;
 import com.concordium.sdk.responses.smartcontracts.ContractTraceElement;
 import com.concordium.sdk.responses.transactionstatus.*;
 import com.concordium.sdk.transactions.*;
 import com.concordium.sdk.types.AccountAddress;
+import com.concordium.sdk.types.ContractAddress;
 import lombok.*;
 
 import java.util.List;
-import java.util.Optional;
 import java.util.stream.Collectors;
 
 /**
  * Details of a transaction with a sender account.
- *
+ * <p>
```

Comment:
:thinking:

Reply:
For getting a newline in the rendered javadoc.